### ISSUE TITLE
npm i fails with EISGIT

### ISSUE THREAD

**Issue by AnastasiiaMarkova24** [perseids-tools/react-router-i18n#19]
When repeatedly performing npm install with a dependency on react-router-i18n, npm fails with the error code EISGIT.

I have declared a dependency on react-router-i18n in my package.json like so:

```
  ...
  "dependencies": {
   "react-router-i18n": "^0.2.0",
    ...
  },
  ...
```
Performing npm install for the first time succeeds. Performing npm install immediately again will fail:

```
npm ERR! code EISGIT
npm ERR! path ...\node_modules\react-router-i18n
npm ERR! git ...\node_modules\react-router-i18n: Appears to be a git repo or submodule.
npm ERR! git     ...\node_modules\react-router-i18n
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

**Comment by zfletch** [comment]
Thanks for pointing out this issue @AnastasiiaMarkova24. It looks like this was caused by a regression in npm version 6.9.0. (See [discussion](https://npm.community/t/npm-6-9-1-is-broken-due-to-git-folder-in-published-tarball/8454/3).)

I've updated npm and republished the package. Version `v0.2.1` should no longer have this issue. Let me know if it still isn't working after this update and I will reopen the issue.
